README.md

```diff
@@ -227,20 +226,20 @@ process sentences, we need to drop any inline code that is (most likely) to be
 ranked as neutral, or may be miscategorized because the model hasn't been
 trained on that particular language.
 
-We also convert any unicode emojis into their short-hand codes (as described on
-http://www.webpagefx.com/tools/emoji-cheat-sheet/), which make it easier on
+We also convert any Unicode emojis into their short-hand codes (as described at
+http://www.webpagefx.com/tools/emoji-cheat-sheet/), which makes it easier on
```

Comment:
As written it appears that "make" points back to "converting," so either "converting" makeS it easier or the *emojis* make it easier and the comma should disappear.